swagger.json

```diff
@@ -2927,13 +2927,19 @@
                     "type": "integer",
                     "format": "int64",
                     "description": "The number of meters reported by the odometer.",
-                    "example": "60130000"
+                    "example": 60130000
                 },
                 "engineHours": {
                     "type": "integer",
                     "format": "int64",
                     "description": "Total engine hours for the vehicle.",
-                    "example": "1500"
+                    "example": 1500
+                },
+                "fuelLevelPercent": {
+                    "type": "float",
+                    "format": "float64",
+                    "description": "The fuel level of the vehicle as a percentage. (0.0 to 1.0)",
+                    "example": 0.3
```

Comment:
I forgot to mention this in the other PR -- do we want to restrict the precision of this percentage value?

Reply:
Probably, this one happens to be an `int64` on the backend so we are ok, but I think providing 2 decimal places would be a good best practice. 

On a related note, golang doesn't have a `round()` or `toPrecision()`? 😕